### PR TITLE
Fixed Javadoc link

### PR DESCRIPTION
**Pull request by jglick**
Fixes links orphaned by #82. @reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java [hunk 1]
```diff
@@ -23,7 +22,7 @@
 
 /**
  * Extension point to subscribe events from GH, which plugin interested in.
- * This point should return true in {@link #isApplicable(AbstractProject)}
+ * This point should return true in {@link #isApplicable}
```
Comment (jtnord) [inline src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java]: I beleive this is an invalid link would refer to the non-existant field `isApplicable` unless something has changed?

Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java]: `isApplicable` is a method, not a field.

Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java]: Specifically `javadoc:javadoc` with JDK7u80 produces

```html
 This point should return true in <a href="../../../../../org/jenkinsci/plugins/github/extension/GHEventsSubscriber.html#isApplicable(hudson.model.Job)"><code>isApplicable(hudson.model.Job&lt;?, ?&gt;)</code></a>
```

as expected.

Comment (jtnord) [inline src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java]: That was my point - I thought you needed to add `(argumentType)` for links to methods. 
I was mistaken.

**Comment by jtnord** [comment]
:bee:

**Comment by oleg-nenashev** [comment]
:bee: . BTW a little :ant: for a risk of Javadoc ambiguous method links if somebody introduces a new isApplicable() method

**Comment by jglick** [comment]
Yes, it is a risk. On the other hand there is a risk of a link breaking if someone changes the type of a parameter, as happened here. :-)

**Comment by lanwen** [comment]
thx!

**Pull request merged**